backup.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/sh
+
+set -e -x
+
+export RUN_MODE=`basename "$0"`
+
+for i in "$@"
+do
+case $i in
+--cloud=*)
+  export CLOUD="${i#*=}"
+  shift
+  ;;
+--region=*)
+  export REGION="${i#*=}"
+  shift
+  ;;
+--bucket=*)
+  export BUCKET="${i#*=}"
+  shift
+  ;;
+--endpoint=*)
+  export ENDPOINT="${i#*=}"
+  shift
+  ;;
+--backup-dir=*|--srcDir=*)
+  export BACKUP_DIR="${i#*=}"
+  shift
+  ;;
+--destDir=*)
+  export DEST_DIR="${i#*=}"
+  shift
+  ;;
+  *)
+  common::log "unknown option [$i]"
```

Comment:
Where is common::log defined?

Reply:
I've removed this function because it seems like `sh` does not support function, but I forget to change this line😅

Reply:
`sh` support function, but does not support your function name `common::log`，this function name not a valid identifier. Eligible function name probably satisfy this regular expression `[a-zA-Z_]{1,}[a-zA-Z0-9_]{0,}`.